src/backend/launcher.ts

```diff
@@ -975,6 +977,63 @@ async function getWinePath({
   return stdout.trim()
 }
 
+/**
+ * Converts a LegendaryCommand to a parameter list passable to Legendary
```

Comment:
can we move this function into some legendary specific file/folder?

Reply:
I anticipate that I'm going to work on applying this same treatment to gogdl and Nile commands in the future, and then I'll need the function here, but for now it can live in Legendary's library
cefadad9e6e56d808cd8a8b75b995fa05f2e38e2